Formula/bitcoind.rb

```diff
@@ -0,0 +1,49 @@
+class Bitcoind < Formula
+  desc "Decentralized, peer to peer payment network"
+  homepage "https://bitcoin.org/"
+  url "https://github.com/bitcoin/bitcoin/archive/v0.14.1.tar.gz"
+  sha256 "4391dbf8fa9683f17c3b03feac429c1f3d71dcc6c0dab7d01733519880ea9834"
+
+  head do
+    url "https://github.com/bitcoin/bitcoin.git"
+  end
+
+  option "with-gui", "Build the GUI client (requires Qt5)"
+
+  depends_on :macos => :lion
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on :xcode => :build
```

Comment:
Does it really require the full Xcode.app to build or are the command line tools sufficient?